roles/openshift_cli/tasks/main.yml

```diff
@@ -28,13 +28,13 @@
 - block:
   - name: Pull CLI Image
     command: >
-      atomic pull --storage ostree {{ 'docker:' if openshift.common.system_images_registry == 'docker' else openshift.common.system_images_registry + '/' }}{{ openshift.common.cli_image }}:{{ openshift_image_tag }}
+      atomic pull --storage ostree {{ 'docker:' if system_images_registry == 'docker' else system_images_registry + '/' }}{{ openshift.common.cli_image }}:{{ openshift_image_tag }}
```

Comment:
If I'm following this right, `system_images_registry` is set to either "docker.io" or "registry.access.redhat.com".  The conditional here then tests if `system_images_registry == 'docker'` which would seem to never evaluate to true.  I was checking CI logs and it appears this code branch is not tested.  Has this been confirmed to be working?  Same for all the other related tasks using this conditional.

Reply:
Yeah, I had the same thought.  I left it as-is in case there's a user out there somewhere that is using it this way.